infra/tpu-ci/modules/arc-v4-container-cluster/main.tf

```diff
@@ -0,0 +1,97 @@
+provider "google" {
+  project = var.project_id
+}
+
+provider "helm" {
+  kubernetes {
+    host = "https://${google_container_cluster.arc_v4_cluster.endpoint}"
+    token = data.google_client_config.default.access_token
+    cluster_ca_certificate = base64decode(google_container_cluster.arc_v4_cluster.master_auth.0.cluster_ca_certificate)
+  }
+}
+
+data "google_client_config" "default" {}
+
+resource "google_container_cluster" "arc_v4_cluster" {
+  name = var.cluster_name
+  location = "us-central2"
+
+  remove_default_node_pool = true
+  initial_node_count = 1
+
+  release_channel {
+    channel = "RAPID"
+  }
+
+  min_master_version = 1.28
+}
+
+resource "google_container_node_pool" "arc_v4_cpu_nodes" {
+  name = var.cpu_nodepool_name
+  location = "us-central2"
+  cluster = google_container_cluster.arc_v4_cluster.name
+  node_count = var.cpu_node_count
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+    ]
+
+    machine_type = "n1-standard-1"
```

Comment:
What does this node do? Just run the GitHub controller? I'm basically asking if 1 CPU is enough.

Reply:
Updated to e2-standard-2